doc/guides/coding_guidelines/index.rst

```diff
@@ -151,6 +151,206 @@ severity and the equivlent rules from other standards for reference.
       -  There shall be no occurrence of undefined or critical unspecified behaviour
       -  N/A
       -  `Rule 1.3 <https://gitlab.com/MISRA/MISRA-C/MISRA-C-2012/Example-Suite/-/blob/master/R_01_03.c>`_
+    * -  Rule 2.1
+      -  Required
+      -  A project shall not contain unreachable code
+      -  `MSC07-C <https://wiki.sei.cmu.edu/confluence/pages/viewpage.action?pageId=87152362/>`_
+      -  `Rule 2.1 <https://gitlab.com/MISRA/MISRA-C/MISRA-C-2012/Example-Suite/-/blob/master/R_02_01.c>`_
```

Comment:
I was clicking through links and noticed that some of the rules have 2-file examples in the MISRA example suite. For example, rule 2.1 corresponds to R_02_01_1.c and R_02_01_2.c, so the link in the documentation to R_02_01.c is broken:

![image](https://user-images.githubusercontent.com/14157493/88215343-1d8d2d80-cc21-11ea-8346-7192d1555522.png)

Directives with broken links: 4.8

Rules with broken links: 2.1, 5.1, 5.8, 5.9, 8.5, 8.6, 13.1, 13.5

(18.6 also has a 2-file example, but it is already fixed)